internal/service/alarms/migration.go

```diff
@@ -0,0 +1,26 @@
+package alarms
```

Comment:
ah good call...flows much nicer by bringing this out to serve.go level. mapping the two commands to the two main files!